superset-frontend/src/views/CRUD/data/dataset/AddDataset/DatasetPanel/DatasetPanel.tsx

```diff
@@ -217,17 +273,23 @@ const DatasetPanel = ({
   return (
     <>
       {tableName && (
-        <StyledHeader
-          position={
-            !loading && hasColumns ? EPosition.RELATIVE : EPosition.ABSOLUTE
-          }
-          title={tableName || ''}
-        >
-          {tableName && (
-            <Icons.Table iconColor={supersetTheme.colors.grayscale.base} />
-          )}
-          {tableName}
-        </StyledHeader>
+        <>
+          {linkedDatasetNames?.includes(tableName) &&
+            renderExistingDatasetAlert(
+              linkedDatasets?.find(dataset => dataset.table_name === tableName),
+            )}
+          <StyledHeader
+            position={
+              !loading && hasColumns ? EPosition.RELATIVE : EPosition.ABSOLUTE
+            }
+            title={tableName || ''}
+          >
+            {tableName && (
+              <Icons.Table iconColor={supersetTheme.colors.grayscale.base} />
```

Comment:
This is where I commented earlier use of supersetTheme directly is an anit pattern we did not catch in initial creation of this file. 

Above in the functional component we should have 
`const theme = useTheme();`

Then here use 
`<Icons.Table` iconColor={theme.colors.grayscale.base} />`

Reply:
FIxed in [`this commit`](https://github.com/apache/superset/pull/22136/commits/86549660bd4d6ac53e0b7723ab3e9370e37d2a28).